src/interface/BaseTooltipProvider.ts

```diff
@@ -20,6 +20,12 @@ class BaseTooltipProvider {
   static itemRelative(id: number, details: any) {
     throw new Error('NotImplemented');
   }
+  static itemSet(...args: [number, any]) {
```

Comment:
This probably needs to be moved into "useTooltips" after the PR was merged earlier today

Reply:
Yeah I saw I conflicted with something as soon as I put up the PR - will do a fix at some point today when I have time